Staging/EnableGop/EnableGop.c

```diff
@@ -0,0 +1,233 @@
+/** @file
+  Provide GOP on unsupported graphics cards on classic MacPro.
+
+  Copyright (c) 2022-2023, Mike Beaton. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-3-Clause
+**/
+
+#include <Uefi.h>
+
+#include <Library/DuetBdsLib.h>
+#include <Library/DxeServicesTableLib.h>
+#include <Library/OcBootManagementLib.h>
+#include <Library/OcConsoleLib.h>
+#include <Library/OcDeviceMiscLib.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+
+#include <Library/OcVariableLib.h>
+
+#define EFI_BLUETOOTH_DELAY_DEFAULT  3000
+
+STATIC EFI_GET_MEMORY_SPACE_MAP  mOriginalGetMemorySpaceMap = NULL;
+
+STATIC
+VOID
+LogStage (
+  CHAR8  Stage
+  )
+{
+  UINT8   One    = 1;
+  CHAR16  Name[] = L"efi-gop-stage0";
```

Comment:
I'd just use ``#define`` - if the optimizer fucks up, there will be a pointless copy on the stack.